alibi_detect/base.py

```diff
@@ -87,14 +87,14 @@ def predict(self, X: np.ndarray):
 
 class FitMixin(ABC):
     @abstractmethod
-    def fit(self, X: np.ndarray) -> None:
-        pass
+    def fit(self, *args, **kwargs) -> None:
+        ...
```

Comment:
I don't mind the `*args, **kwargs` strategy here, but will be good to check with @jklaise next week too.